src/wovnio/modified_vendor/simple_html_dom.php

```diff
@@ -81,7 +81,7 @@ class simple_html_dom_node
 	public $children_list_head = null;
```

Comment:
This file use tab instead of space for indentation. Could you reformat code to use space as in other files?

Reply:
I will do another PR.
Because this PR contains many changes and the style problem of this file is not only space, I want to do another PR for easiness.

problems are:
* class' name (simple_html_dom -> SimpleHtmlDom)
* declaring two class in file